controller/controller.go

```diff
@@ -750,7 +750,7 @@ func saveInstallResults(rootDir string, md *model.SystemInstall) error {
 		log.Info("Skipping archiving of Installation results")
 	}
 
-	if md.IsTelemetryEnabled() {
+	if md.IsTelemetryEnabled() && md.IsTelemetryInstalled() {
```

Comment:
Please change this to have the IsTelemetryInstalled nested inside of the Enabled
```suggestion
	if md.IsTelemetryEnabled()  {
	    if md.IsTelemetryInstalled() {
```